### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by BingyuanW** [aliyun/NeWCRFs#5]
Hi, it seems that the learing rate mentioned in your paper is a constant, but in your training code, the lr :
![image](https://user-images.githubusercontent.com/53251573/164056979-54217f5b-eb82-49dd-bc2d-14f4691388d2.png)
can you explain that ? Looking forward to your reply.

**Comment by weihaosky** [comment]
Yes, the learning rate is decreased in the training. The learning rate mentioned in the paper is the learning rate at the beginning. We do not pay much attention to the learning rate. The results should be similar with a fixed learning rate.
